### PR TITLE
fix broken behavior for LoadTeamsPlusApplicationKeys

### PR DESCRIPTION
**Pull request by maxtaco**
- all writers, owners and admins are writers
- only readers are onlyReaders
- no need for set intersections

### CODE REVIEW

**Comment by maxtaco** [comment]
CC: @modalduality  and @oconnor663  --- are there any tests for this code?  I couldn't find any.

**Comment by maxtaco** [comment]
cc: @strib  who hit the broken behavior and asked for a fix.

File: go/teams/rpc_exim.go [hunk 1]
```diff
@@ -11,36 +11,34 @@ import (
 	keybase1 "github.com/keybase/client/go/protocol/keybase1"
 )
 
-func (t *Team) ExportToTeamPlusApplicationKeys(ctx context.Context, idTime keybase1.Time, application keybase1.TeamApplication) (teamPlusApplicationKeys keybase1.TeamPlusApplicationKeys, err error) {
+func (t *Team) ExportToTeamPlusApplicationKeys(ctx context.Context, idTime keybase1.Time, application keybase1.TeamApplication) (ret keybase1.TeamPlusApplicationKeys, err error) {
 	applicationKeys, err := t.AllApplicationKeys(ctx, application)
 	if err != nil {
-		return
+		return ret, err
```
Comment (oconnor663) [inline go/teams/rpc_exim.go]: If we want to make all the returns explicit, we can unname the return params?

Comment (maxtaco) [inline go/teams/rpc_exim.go]: I usually do it this was since it's less typing, why is it bad?

Comment (oconnor663) [inline go/teams/rpc_exim.go]: Sorry I think I put that in a confusing way. I was suggesting we remove the named returns from the signature of the function. But I definitely don't care very much :)

**Comment by patrickxb** [comment]
LGTM

**Pull request merged**